app/models/user.rb

```diff
@@ -1,2 +1,16 @@
 class User < ApplicationRecord
+    has_one :token
+
+    def rspotify
+        RSpotify::User.new({
+            'id' => spotify_id,
+            'email' => email,
+            'credentials' => {
+                'refresh_token' => token.refresh_token,
+                'token' => token.access_token,
+                'token_type' => token.token_type,
+                'scope' => token.scope,
+            }, 
+        })
+    end
```

Comment:
By adding this in the model, you can now get a user object, and call user.rspotify to get the rspotify user to work with.
```
user = User.where(:spotify_id=>"jingni").first
user.rspotify
```